### ISSUE TITLE
Provide alternate mirror links of articles to archive.is or archive.org

### ISSUE THREAD

**Issue by WeebDataHoarder** [pox/EndTheFUD#9]
Some sources linked (for example medium) require creating an account with them or have a rate limit of how many articles you can view. Sites might stop working, or block visitors due to legal reasons.

Providing mirrors (in addition of original source) would allow preserving the contents and let visitors decide to use these alternative mirrors. Even better, you might be able to have local text archives on repository, but that's a stretch idea.

**Comment by pox** [comment]
Good point.

I think it's ultimately up to the content creators who chose to use Medium. IIRC medium actually own the copyright of the posts, not the authors. At least I'm certain that's the way it used to be.

Regardless, I cannot simply copy those articles. I do consider Medium to be a free source, if only because it's so easy to get around those silly limits (is there any article listed that cannot be view in Incognito mode?).

**Comment by 9mido** [comment]
An archive mechanism would be great because eventually some links will 404 and no longer exist. Maybe you could put the link to the original resource and then next to it put the archive link in case the original disappears.

**Comment by pox** [comment]
You mean an [archive.org](https://archive.org/) link, or a link to the project's own copy? Like I mentioned, I'm not sure it's cool/legal/ethical to copy the original article. 

Might be best to keep it simple ™ and add archive.org links as they're needed, if and when they're needed.

**Comment by 9mido** [comment]
Yes something like that. Understood. Do whatever you think is best.
